en/overview/webworldwind_overview.rst

```diff
@@ -22,13 +15,9 @@ Web World Wind
 Virtual Globe in a browser
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. Cameron comment: there is overlap between introduction setence and following
-  The statement about Web World Wind use is the same for both Java World Wind
-  and Web World Wind. (It is a good statement). But it looks like a
-  cut-and-paste, which reduces its credibility. Better to select different use
-  cases for each.
 
-Web World Wind is a browser based, 3 dimensional virtual globe. It allows users to zoom from satellite altitude to any place on earth, experiencing the earth in 3 dimensions, as if really there. It is easy to use, only requiring a two button mouse.  It is used to monitor weather patterns, visualize cities and terrain, track the movement of planes, vehicles and ships, analyze geospatial data, and educate people about the Earth.
+Web World Wind is a browser based, 3 dimensional virtual globe. It allows users to zoom from satellite altitude to any place on earth, experiencing the earth in 3 dimensions, as if really there. 
+It is easy to use, only requiring a two button mouse.  It is used to monitor weather patterns, visualize cities and terrain, track the movement of planes, vehicles and ships, analyze geospatial data, and educate people about the Earth.
 
 Web World Wind provides an extensible, javascript platform which enables web developers to quickly create custom globes and 2 dimensional maps which can express, manipulate and analyze spatial data.  You can show terrain and high-resolution imagery, include a compass, map coordinates, and view controls.
 

```

Comment:
The Web World Wind picture of a globe looks good, but there is a lot of white space in the image which is a bit of a waste. There are white strips on both left and right. The "World Wind Basic Example" title is in a very large font and not adding much value. I suggest seeing if you could get a better screen shot.